macros/Compat.ejs

```diff
@@ -698,8 +698,9 @@ if (!compatData) {
 traverseFeatures(compatData, depth, '');
 
 if (features.length > 0) {
+    const childPath = query.split(".").join("/") + ".json";
     output = `<div class="bc-data" id="bcd:${query}">`;
-    output += '<a class="bc-github-link" href="https://github.com/mdn/browser-compat-data">Update compatibility data on GitHub</a>';
+    output += `<a class="bc-github-link" href="https://github.com/mdn/browser-compat-data/${childPath}">Update compatibility data on GitHub</a>`;
```

Comment:
Pardon my ignorance, but doesn't that require the `blob/master` part of the URL also?

I'd rather the "source map" was something that was built in rather than relying on and hoping the `query` maps to the file in the "git browser" (aka. GitHub web UI).

Reply:
I thought that link to `edit/master` is better than `blob/master`, and reflected it in [37e547b](https://github.com/mdn/kumascript/pull/1347/commits/37e547b3745ebf763489acfb6793d9185c3ca25f). But I'm unsure if that was what you thought.